ee/dtr/index.md

```diff
@@ -6,61 +6,41 @@ keywords: registry, repository, images
 
 >{% include enterprise_label_shortform.md %}
 
-Docker Trusted Registry (DTR) is the enterprise-grade image storage solution
-from Docker. You install it behind your firewall so that you can securely store
-and manage the Docker images you use in your applications.
+Docker Trusted Registry (DTR) is Mirantis's enterprise-grade image storage solution. Installed behind the firewall, either on-premises or on a virtual private cloud, DTR provides a secure environment from which users can store and manage Docker images.
 
-## Image and job management
+Specifically, DTR offers a wide range of benefits that include:
 
-DTR can be installed on-premises, or on a virtual private
-cloud. And with it, you can store your Docker images securely, behind your
-firewall.
+* [Image and Job Management](image-and-job-management)
+* [Availability](availability)
+* [Efficiency](efficiency)
+* [Built-in Access Control](built-in-access-control)
+* [Security Scanning](security-scanning)
+* [Image Signing](image-signing)
 
-You can use DTR as part of your continuous integration, and continuous
-delivery processes to build, ship, and run your applications.
+## Image and Job Management
+ DTR offers a web-based user interface with which users can browse Docker images and [review repository events](/ee/dtr/user/audit-repository-events/). With the UI, in fact, users can even see which Dockerfile lines were used to produce an image and, if security scanning is enabled, a list of all of the software installed in that image. In addition, the web UI can be used to [review and audit jobs](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/).
```

Comment:
s/user interface/UI

Reply:
"DTR has a web-based user interface that you can use to browse images, [audit repository events](/ee/dtr/user/audit-repository-events/). You can see which lines in the Dockerfile produced the image, and if security scanning is enabled, you can see a list of the software installed on the image. "

Not sure you need the last sentence but if it's important can you shorten it to something like "You can also [audit jobs in the web interface](/ee/dtr/admin/manage-jobs/audit-jobs-via-ui/)."

If you have to overwrite the title of an article that you're linking to, maybe the article title should change. It's ok to do this once in a while but it's confusing when the link text and the title of the topic are not the same. 
And via? Don't get me started.

Reply:
Feedback integrated. Thanks.